src/common/commandExecutor.ts

```diff
@@ -96,33 +94,32 @@ export class CommandExecutor {
             }
         });
 
-        Q.delay(300).then(() => deferred.resolve(result.spawnedProcess));
+        // TODO - PROMISE: We need to consume result.outcome here
```

Comment:
I think that this code will become unnecessary after we fix: https://github.com/Microsoft/vscode-react-native/issues/56 , so I'm waiting for that fix